precompiles/parachain-staking/src/lib.rs

```diff
@@ -160,15 +158,9 @@ where
 			Action::ScheduleCandidateBondLess => {
 				Self::schedule_candidate_bond_less(input, context)?
 			}
-			// DEPRECATED
```

Comment:
should these still be deprecated or were they not supposed to be deprecated in the first place?

Reply:
This is no longer deprecated because candidate_bond_more name and behavior did not change